testing/tests/DevExpress.ui.widgets.dataGrid/columnsResizingReorderingModule.tests.js

```diff
@@ -4342,7 +4366,12 @@ QUnit.module('Headers reordering', {
         const controller = this.createDraggingHeaderViewController();
         const draggingHeader = new TestDraggingHeader(this.component);
 
-        $('#itemsContainer').html('<div style="width:125px; display: inline-block;" /><div style="width:125px; display: inline-block;" />');
+        $('#itemsContainer').html('<div /><div />');
+
+        $('#itemsContainer > div').css({
+            width: '125px',
+            display: 'inline-block'
+        });
```

Comment:
Let's just add classes here, instead of setting CSS rules by hand with JS:

```suggestion
        $('#itemsContainer').html(`
          <div class="itemsContainer__child" ></div>
          <div class="itemsContainer__child" ></div>
        `);
```